packages/contracts/src/contract-deployment/config.ts

```diff
@@ -192,59 +105,16 @@ export const makeContractDeployConfig = async (
         config.ovmGlobalContext,
       ],
     },
-    OVM_StateManager: {
```

Comment:
Whoops my apologies, this contract should actually still be included.

Reply:
@smartcontracts no worries, I included the contract in the latest commit.